src/index.html

```diff
@@ -11,7 +11,192 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form action="#" method="get">
+      <fieldset>
+        <legend>Personal information</legend>
+        <div> Surname:
+          <label for="surname">
+            <input
+              id="surname"
+              type="text"
+              name="surname"
+              maxlength="20"
+            >
+          </label>
+        </div>
```

Comment:
Surname string should be inside tag label
```suggestion
        <div>
          <label for="surname">
            Surname:
            <input
              id="surname"
              type="text"
              name="surname"
              maxlength="20"
            >
          </label>
        </div>
```